support/cas-server-support-gauth-core-mfa/src/main/java/org/apereo/cas/gauth/validator/GoogleAuthenticatorAccountValidator.java

```diff
@@ -0,0 +1,30 @@
+package org.apereo.cas.gauth.validator;
+
+import org.apereo.cas.authentication.OneTimeTokenAccount;
+import org.apereo.cas.otp.validator.OneTimeTokenAccountValidator;
+
+import com.warrenstrange.googleauth.IGoogleAuthenticator;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * This is {@link GoogleAuthenticatorAccountValidator}.
+ *
+ * Implements {@link OneTimeTokenAccountValidator} to
+ * provide validation of Google Authenticator credentials.
+ *
+ * @author Hayden Sartoris
+ * @since 6.2.0-RC5
```

Comment:
Remove the reference to RC

Reply:
639eba8179badc3b14ca8966cc87c557b3f75167